pkg/arvo/sys/vane/clay.hoon

```diff
@@ -4828,6 +4828,8 @@
       %merg                                               ::  direct state up
     ?:  =(%$ des.req)
       ~&(%merg-no-desk !!)
+    ?.  ((sane %tas) des.req)
+      ~&(%merg-bad-desk-name !!)
```

Comment:
Consider including the desk name in the print, and making the print part of the crash trace so it renders more obviously as the cause of the crash.

```suggestion
      ~|([%merg-bad-desk-name des.req] !!)
```